src/thunderbots/software/geom/angle.h

```diff
@@ -575,9 +576,17 @@ inline constexpr bool operator>=(Angle x, Angle y)
     return x.toRadians() >= y.toRadians();
 }
 
-inline constexpr bool operator==(Angle x, Angle y)
+inline bool operator==(Angle x, Angle y)
 {
-    return x.toRadians() == y.toRadians();
+    // Due to internal representation of doubles being slightly less accurate/consistent
+    // with some numbers and operations, we consider angles that are very close together
+    // to be equal (since they likely are, just possibly slightly misrepresented by the
+    // system/compiler). 1e-15 was chosen as a value because doubles have about 16 digits
+    // of precision. Comparing numbers with 15 digits of precision gives us a small buffer
+    // while remaining as accurate as possible.
+    // http://www.cplusplus.com/forum/beginner/95128/
+    double EPS = 1e-15;
```

Comment:
We should probably expand `EPS` to `EPSILON`, just to be a bit more clear. Also, this seems like it should be something that should be associated with the `Angle` class, rather then this function. Maybe a static class member?